src/Mod/Arch/ArchComponent.py

```diff
@@ -741,22 +1050,56 @@ def isStandardCase(self,obj):
 
 
 class ViewProviderComponent:
+    """A default View Provider for Component objects.
 
-    "A default View Provider for Component objects"
+    Acts as a base for all other Arch view providers. It’s properties and
```

Comment:
Here you are adding a non-ASCII character to the docstring. The quote character is not a simple quote `'`, but it's a typographical quotation symbol `’`, like if you wrote this with a word processor. Instead of adding utf8 codification to the file, it's best if you just remove this symbol, and use a simple quote `'`.

Reply:
Resolved in 0feb9b8ab268ecca9352d4f598f12ab066ff207e